pallets/loans/src/tests.rs

```diff
@@ -479,14 +479,14 @@ fn test_price_bullet_loan() {
 				math::seconds_per_year() * 2,
 			));
 			let rp = Zero::zero();
-			let res = Loans::price(
+			let _res = Loans::price(
 				RuntimeOrigin::signed(borrower),
 				pool_id,
 				loan_id,
 				rp,
 				loan_type,
 			);
-			assert_err!(res, pallet_interest_accrual::Error::<Runtime>::InvalidRate);
+			//assert_err!(res, pallet_interest_accrual::Error::<Runtime>::InvalidRate);
```

Comment:
These tests are legacy from `pallet-loans` and now are failing because we allow `Zero` rates in the `interest-accrual` pallet. If we do not want to price with a zero rate, should be `price()` the entity in charge of checking this.

Because this will be deprecated I'm not changing anything.